### PR TITLE
Added the scholarships page

### PR DESCRIPTION
**Pull request by olasitarska**
Feedback on this welcome!

### CODE REVIEW

File: templates/scholarship.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    </p>
+    <p>
+    	The application process is simple: Tell us about what you do with Python or Django, and why you want to attend Django: Under The Hood.
+    </p>
+
+    <p>
+    	<b>Scholarships are made possible by our <a href="/#sponsors">generous sponsors</a>. We are very proud to be part of this community for showing their support. Thank you!</b>
+    </p>
+
+    <p style="text-align: center; margin: 30px 0">
```
Comment (bmispelon) [inline templates/scholarship.html]: What's up with the inline CSS?

File: templates/scholarship.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    </p>
+    <p>
+    	Anyone from an underrepresented group in tech is invited to apply for this scholarship. This includes, but is not limited to: women, people of colour, LGBTQIA+ people, disabled people, and people facing economic or social hardships.
+    </p>
+    <p>
+    	The application process is simple: Tell us about what you do with Python or Django, and why you want to attend Django: Under The Hood.
+    </p>
+
+    <p>
+    	<b>Scholarships are made possible by our <a href="/#sponsors">generous sponsors</a>. We are very proud to be part of this community for showing their support. Thank you!</b>
```
Comment (bmispelon) [inline templates/scholarship.html]: Can you use `{% url 'home' %}#sponsors`?

File: templates/scholarship.html [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+{% load staticfiles %}
+
+{% block content %}
+
+<section id="coc" class="row">
+    <h1>
+        Scholarships for Django: Under The Hood!
+    </h1>
+    <p>
+    	Django: Under The Hood is striving to provide a great experience for the most diverse group of people possible. We do believe in equality, and we invite everyone to participate. In order to make this possible, we are proud to announce our scholarship programme for people who would otherwise be unable to participate.
```
Comment (bmispelon) [inline templates/scholarship.html]: `<strong>scholarship programme</strong>`?

File: templates/scholarship.html (hunk 1 above)
Comment (olasitarska) [inline templates/scholarship.html]: Ah yeah I was experimenting and then forgot to move it to css file :)

File: templates/scholarship.html [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    	<b>Scholarships are made possible by our <a href="{% url 'home' %}#sponsors">generous sponsors</a>. We are very proud to be part of this community for showing their support. Thank you!</b>
+    </p>
+
+    <p class="centered-margin">
+		<a class="btn" href="http://goo.gl/forms/9Ct9jQCwDf">Apply now for the Django: Under The Hood Scholarship</a>
+    </p>
+
+    <p>Applications are due 21st August, by the end of the day CEST (Amsterdam time).</p>
+
+    <p>Thank you <a href="http://2015.jsconf.eu/">JSConf EU</a> for providing a great example of the proccess we're adopting.</p>
```
Comment (erikr) [inline templates/scholarship.html]: proccess -> proces

Comment (olasitarska) [inline templates/scholarship.html]: you sure it's not process? proces sounds awfully Polish ;)

File: duh/views.py [hunk 5]
```diff
@@ -25,3 +25,6 @@ def attendees(request):
 
 def talks(request):
     return render(request, 'talks.html', {})
+
+def scholarship(request):
+    return render(request, 'scholarship.html', {})
```
Comment (mjtamlyn) [inline duh/views.py]: Newline at EOF

Comment (olasitarska) [inline duh/views.py]: Done ;)

File: templates/scholarship.html (hunk 4 above)
Comment (mjtamlyn) [inline templates/scholarship.html]: It's process (one c, two s). So Both of Erik's are wrong ;)

Comment (olasitarska) [inline templates/scholarship.html]: aha! fixed.

Comment (erikr) [inline templates/scholarship.html]: Proces is Dutch, I thought we were doing a Dutch-only website ;)

**Comment by williln** [comment]
> people with impairments

Suggest replacing with, "people with disabilities." 

>We are very proud to be part of this community for showing their support.

This phrasing is a little awkward; maybe something like, "We are proud to have these sponsors showing their support for our community." 

>Apply now for the Django: Under The Hood Scholarship 

Toward the bottom. This is picky, but I'd say "Apply now for a ..." "The" sounds like there is only one scholarship; "a" scholarship implies that there are multiple.

**Comment by olasitarska** [comment]
Thanks @williln! :blossom:

**Pull request merged**